pkg/alertmanager/amcfg_test.go

```diff
@@ -1004,8 +803,16 @@ func TestInitializeFromAlertmanagerConfig(t *testing.T) {
 				t.Errorf("initializeFromAlertmanagerConfig() error = %v, wantErr %v", err, tt.wantErr)
```

Comment:
nit: looks like we don't need this now as we are using require.Error in next step